components/ds18x20/schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "title": "DS18X20 Component Definition",
+  "description": "A DS18X20 WipperSnapper component for use in Adafruit IO",
+  "type": "object",
+  "required": [ "displayName", "sensorType", "sensorResolution"],
+  "properties": {
+    "displayName": {
+      "description": "The human-friendly name of this component.",
+      "type": "string",
+      "minLength": 3,
+      "maxLength": 24
+    },
+    "sensorType": {
+        "description": "One of the supported sensor type strings (found in README).",
+        "type": "string",
+        "pattern": "^(unspecified|accelerometer|magnetic-field|orientation|gyroscope|light|pressure|proximity|gravity|acceleration|rotation|humidity|ambient-temp|object-temp|voltage|current|color|raw|pm10-std|pm25-std|pm100-std|pm10-env|pm25-env|pm100-env|co2|altitude|gas-resistance|tvoc)$"
+    },
+    "sensorResolution": {
+        "description": "The DS18X20's desired sensor read resolution, in bits.",
+        "type": "number",
+        "minimum": 9,
+        "maximum": 12
+    },
```

Comment:
Is the user able to set this in the form?

Reply:
They should be able to set this in the form.